src/Sylius/Bundle/ApiBundle/ApiPlatform/ApiResourceConfigurationMerger.php

```diff
@@ -13,6 +13,13 @@
 
 namespace Sylius\Bundle\ApiBundle\ApiPlatform;
 
+trigger_deprecation(
+    'sylius/api-bundle',
+    '1.14',
+    'The "%s" class is deprecated and will be removed in Sylius 2.0.',
+    ApiResourceConfigurationMerger::class,
+);
+/** @deprecated since Sylius 1.14 and will be removed in Sylius 2.0. */
```

Comment:
```suggestion
);

/** @deprecated since Sylius 1.14 and will be removed in Sylius 2.0. */
```
and the same would be applied to other classes